tests/oss-fuzz/avif_decode_fuzzer.cc

```diff
@@ -18,6 +18,11 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t * Data, size_t Size)
     static size_t yuvDepthsCount = sizeof(yuvDepths) / sizeof(yuvDepths[0]);
 
     avifDecoder * decoder = avifDecoderCreate();
+    // ClusterFuzz passes -rss_limit_mb=2560 to avif_decode_fuzzer. Empirically setting
+    // decoder->imageSizeLimit to this value allows avif_decode_fuzzer to consume no more than
+    // 2560 MB of memory.
+    static_assert(11 * 1024 * 10 * 1024 <= AVIF_DEFAULT_MAX_IMAGE_SIZE, "");
+    decoder->imageSizeLimit = 11 * 1024 * 10 * 1024;
```

Comment:
We will need to set decoder->imageSizeLimit to an even smaller value to prevent avif_decode_fuzzer from timing out in the YUV<->RGB conversion tests. But I haven't done the experiments to figure out what that value should be.

Alternatively, we can add a dedicated test to call the YUV<->RGB conversion functions with all kinds of sizes and options, and remove the YUV<->RGB conversion tests from avif_decode_fuzzer.